utils/mstpctl-utils-functions.sh

```diff
@@ -2,7 +2,7 @@ mstpctl_parse_ports()
 {
   while [ x"${1+set}" = xset ]
   do
-    # For compatibility: the `all' option.
+    # For compatibility: the 'all' option.
```

Comment:
ummm , i'm unclear about what you mean by "we are forking the development of this file"